doc/emergency-banner.md

```diff
@@ -0,0 +1,19 @@
+# Emergency banner
+
+See the [opsmanual](https://docs.publishing.service.gov.uk/manual/emergency-publishing.html#adding-emergency-publishing-banners) for information about what the Emergency Banner is and when it should be deployed.
+
+## Running the Emergency Banner locally
+
+### Deploying the Emergency banner
+
+```
+bundle exec rake emergency_banner:deploy['{campaign_class}','{heading}','{short_description}','{link}']
+```
+
```

Comment:
Not sure if it's within the scope of your work but it'd be good to rename the classes so they're based semantic rather than based on their presentation.

e.g.

'red => alert'
'black => fatality'
'green => (not sure what this means)'